src/utils/attributes-normalizer.js

```diff
@@ -15,7 +15,7 @@ function coerce(string) {
   // Complex object
   try {
     val = JSON.parse(string);
-  } catch (e) {
+  } catch (e) { // eslint-disable-line no-unused-vars
```

Comment:
I would console.warn the error instead of the eslint disable line ... since anyway we have other console.warn statements .... later we need to start thinking how we should report these errors somewhere

Reply:
In principle I agree, but it seems like in this case the author used try...catch to silence the failing `JSON.parse()`
I'll try thinking of a better way to write this